install/kubernetes/helm/subcharts/prometheus/values.yaml

```diff
@@ -7,6 +7,9 @@ hub: docker.io/prom
 tag: v2.3.1
 retention: 6h
 
+# Controls the frequency of prometheus scraping
+scrapeInterval: 15s
+
```

Comment:
Does it make sense to `defaultScrapeInterval` and `istioScrapeInterval`
One for istio jobs and one for application / envoy jobs ?

Reply:
Maybe. I was trying to simplify (and avoid configuration overload), but I have no strong feelings either way here.

Reply:
I suggest waiting on this until a need arises.